BackofficeBundle/Resources/public/less/smartadminPatches/modal.less

```diff
@@ -8,8 +8,19 @@
         height: 90%;
         width: 90%;
 
+        .modal-header {
+            overflow: auto;
```

Comment:
i think it's not a good idea to fix height and add an overflow, you can use instead a clearfix for manage the float